### ISSUE TITLE
make possible to define which Filter parameters should be build into the URL path or query list

### ISSUE THREAD

**Issue by tristanlins** [MetaModels/core#139]

**Comment by xantippe** [comment]
=> was ist damit gemeint?

**Comment by discordier** [comment]
War ein request von mir auf der Konferenz 2013.
Es geht darum dass man Filter Parameter auch als Query string konfigurieren koennen sollte.
Also: `products/<category>?year=<year>`
Hierbei ist die Kategorie Teil der URL und das Jahr Teil des Query string.
Das erfordert aber deutlichen Umbau der internen API. Daher noch kein Milestone in Sicht.

**Comment by xantippe** [comment]
o.k. - soll bedeuten, dass "category" in die URL "kommt" ohne dass es ein zugehöriges Eingabefeld im FE-Filter gibt..?

**Comment by discordier** [comment]
Nein, das soll bedeuten, dass man definieren kann wo die filter Werte in der URL landen sollen.
Eben ob als teil des paths oder aber als teil des Querystring

**Comment by xantippe** [comment]
also man kann/könnte z.B. definieren, dass category in den path und year in den qyery kommt - like
products/category/icecream.html?allergenic=nuts

**Comment by discordier** [comment]
Genau das.

**Comment by stefanheimes** [comment]
Umsetzung: 
- Anpassen des Simple Filters 
- Hinzufügen der Auswahl ob Slack oder GET in den Filter Einstellungen/Settings/Rules
- Anpassen des DCA für die Ausgabe
- Anpassen der URL Builder Funktionen
- Anpassen der Auswertung (Nötig? Da Contao eh alles auf GET Überträgt)

**Comment by zonky2** [comment]
fixed
